server/repository/src/migrations/v1_01_03/mod.rs

```diff
@@ -0,0 +1,45 @@
+use crate::StorageConnection;
```

Comment:
At the moment there are two ways to do migrations. 
1) The diesel migrations which only support SQL queries
2) These migration which allow us to run arbitrary code with a migration

So please remove either this or the diesel migrations... Personally I would go with the diesel migration because than you avoid potential merge conflicts with other V1_01_03 migrations...